src/components/InstallationPage/index.jsx

```diff
@@ -63,11 +63,13 @@ const InstallationPage = function() {
   const { t } = useI18n()
 
   const bitwardenSettings = useContext(BitwardenSettingsContext)
+  const isVaultConfigured =
```

Comment:
Nit: I'm not a big fan of renaming the variable. Why not using : isExtensionInstalled ? Does the vault is really configured if the extension is installed? Because I can have the extension installed but not logged in my cozy yet, right?

Reply:
This is extension_installed that is misnamed :( Extension_installed means that either an extension or an app has been connected to the cozy.